plugins/kotlin/j2k/k1.new/src/org/jetbrains/kotlin/nj2k/conversions/RemoveWrongExtraModifiersConversion.kt

```diff
@@ -6,14 +6,23 @@ import org.jetbrains.kotlin.nj2k.NewJ2kConverterContext
 import org.jetbrains.kotlin.nj2k.RecursiveConversion
 import org.jetbrains.kotlin.nj2k.tree.*
 
-internal class RemoveWrongExtraModifiersForSingleFunctionsConversion(context: NewJ2kConverterContext) : RecursiveConversion(context) {
+internal class RemoveWrongExtraModifiersConversion(context: NewJ2kConverterContext) : RecursiveConversion(context) {
```

Comment:
I guess we should move here all logic from the code builder about redundant modifiers. I can do it later, it's not necessary for this PR.

Reply:
On the other hand, the cases are a little bit different. In this conversion, we can just delete "other" modifiers. But it doesn't work for modality and visibility: we can't really delete them, we can only "skip" printing them. So, I think I will just extract the logic out of the code builder into a helper function.